### PR TITLE
Project modal

### PR DESCRIPTION
**Pull request by sediqullahbadakhsh**
With respect to the Figma Template the following functionality added to Project section (not all cards):
1- created modal section within html (there are 2 but It was for expermenting for all cards which I am working on it and I guess it is not part of the requirements :).
2- add styles for the Mobile first
3- created responsivness for both Tablet and Desktop something similar to Figma.
4- applied Popup functionality using JavaScript.
5- add close btn funtionality to the modal.
 
Edite:

thanks for the nice feedback, I made the following changes as per request:
1- created dynamic modals using JavaScript object for the cards (only 6 cards as the project already had the popup window).
2- added Icons
3- changed the font stylings.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -334,6 +351,83 @@ <h6 id="text">
...
+                electronic typesetting, remaining essent
+              </p>
+              <ul class="modal-btn">
+                <li>
+                  <button class="project-button" type="button">See Live</button>
+                </li>
+                <li>
+                  <button class="project-button" type="button">
+                    See source
+                  </button>
```
Comment (ShoiraTa) [inline index.html]: - [ ] Please add icons on buttons as provided in Figma:

Figma  | Yours
------------- | -------------
image | image

File: index.html [hunk 2]
```diff
@@ -334,6 +351,83 @@ <h6 id="text">
...
+            />
+            <div class="modal-details">
+              <p>
+                Lorem Ipsum is simply dummy text of the printing and typesetting
+                industry. Lorem Ipsum has been the industry's standard dummy
+                text ever since the 1500s, when an unknown printer took a galley
+                of type and scrambled it to make a type specimen book. It has
+                survived not only five centuries, but also the leap into
+                electronic typesetting, remaining essent
+              </p>
```
Comment (ShoiraTa) [inline index.html:365]: - [ ] Please set the typography style as provided in Figma, you can check the corresponding styles on the right side menu:

image
You would have to change line height, font color etc.

File: app.js [hunk 3]
```diff
@@ -13,3 +13,14 @@ const navSlide = () => {
 };
 
 navSlide();
+const modalBtn = document.querySelector('.project-button');
+
```
Comment (ShoiraTa) [inline app.js:18]: - [ ] Based on project requirements, you should have a JS object with all the project data that populates the popup modal:

image
 
Please add the JS object with project data.

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi team,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by t-yanick — APPROVED** [review]
Hi @sediqullahbadakhsh, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**